sensor.py

```diff
@@ -106,3 +123,95 @@ def icon(self):
     @property
     def device_class(self):
         return DEVICE_CLASS_BATTERY
+
+
+class HSTemperature(HSSensor):
+    """Representation of a HomeSeer temperature sensor device."""
+
+    @property
+    def unit_of_measurement(self):
+        if self._units is not None:
+            if str(self._units) == 'imperial':
+                return "°F"
+            elif str(self._units) == 'metric':
+                return "°C"
+            else:
+                return "?"
+        else:
+            return '?'
```

Comment:
What we need to figure out here is what unit the data comes from HomeSeer in - i.e. is value sent over the API in Celsius or Fahrenheit? Home Assistant will handle the conversion/representation in the frontend etc automatically depending on what localization option you select.

Reply:
The values coming from HS do not have units and there isn't anything in the API documentation that talks about a way to get the unit setting from HS.  I made the assumption that the unit system selected in Home Assistant would likely match the unit system selected in HomeSeer.

Reply:
I guess my question is, are the values from the API always in the same unit, or do they change depending on what the user has selected in HomeSeer. Ecobee, for example, always send Fahrenheit from their API regardless of what localization settings are selected for the account. Makes it easier to automate. 

If the value over the HS API is not consistent, then the way to solve this is through a configuration.yaml config key. We should not make assumptions for the user.